chart/templates/configmap-env.yaml

```diff
@@ -31,8 +31,13 @@ data:
   MALLOC_ARENA_MAX: "2"
   NODE_ENV: "production"
   RAILS_ENV: "production"
+  {{- if .Values.redis.enabled }}
   REDIS_HOST: {{ template "mastodon.redis.fullname" . }}-master
   REDIS_PORT: "6379"
```

Comment:
isn't this already covered by both `{{ .Values.redis.port | default "6379" | quote }}` and also the new `redis.port` defaulting to 6379?

Reply:
It is, and I have to apologize  😣 I made a comment yesterday based on the current changes on `main` but after taking a closer look at @norman-zon's code I realized that his implementation was better. I deleted the message soon afterwards hoping no one had seen it, but I guess it made it to some mailboxes.